webpack.config.test.cjs

```diff
@@ -37,20 +38,27 @@ function getConfig(group, { coverage = false } = {}) {
       ...webpackConfig,
       // Karma watches the test entry points, so we don't need to specify
       // them here. Webpack watches dependencies.
-      entry: null,
+      entry: undefined,
       mode: 'development',
       devtool: 'inline-source-map',
       output: {
         ...webpackConfig.output,
         path: path.resolve(process.cwd(), 'assets', 'testjs'),
       },
-      // DependencyExtractionWebpackPlugin and HtmlWebpackPlugin are not needed for tests and
-      // otherwise has some failures.
-      plugins: webpackConfig.plugins.filter(
-        (plugin) =>
-          !(plugin instanceof DependencyExtractionWebpackPlugin) &&
-          !(plugin instanceof HtmlWebpackPlugin)
-      ),
+      plugins: [
+        // DependencyExtractionWebpackPlugin and HtmlWebpackPlugin are not needed for tests and
+        // otherwise has some failures.
+        ...webpackConfig.plugins.filter(
+          (plugin) =>
+            !(plugin instanceof DependencyExtractionWebpackPlugin) &&
+            !(plugin instanceof HtmlWebpackPlugin)
+        ),
+        // React Testing Library checks for this variable, but webpack does
+        // no longer polyfill `process` in the browser.
+        new webpack.DefinePlugin({
+          'process.env.RTL_SKIP_AUTO_CLEANUP': false,
+        }),
```

Comment:
Not needed anymore with v11.2.7+

See https://github.com/testing-library/react-testing-library/pull/911